xc7/make/project_xray.cmake

```diff
@@ -47,9 +47,34 @@ function(PROJECT_XRAY_DUMMY_SITE)
 endfunction()
 
 function(PROJECT_XRAY_TILE)
+  #
+  # This function is used to create targets to generate pb_type, model and tile XML definitions.
+  #
+  # PART name of the part that is considered (e.g. artix7, zynq7, etc.)
+  # TILE name of the tile that has to be generated (e.g. CLBLM_R, BRAM_L, etc.)
+  # SITE_TYPES list of sites contained in the considered tile (e.g. CLBLM_R contains a SLICEM and SLICEL sites)
+  # EQUIVALENT_TILES list of tiles equivalent to the considered one (e.g. CLBLL_R is equivalent to CLBLM_R and CLBLM_L)
+  # PIN_PREFIX translation pattern between prefixes of two equivalent tiles. This is needed as CLBLLs and CLBLMs have different
```

Comment:
When the tile split is done, no prefix translation will be required.  How will that be expressed?

Reply:
I have deleted the PIN_PREFIX, given that the prefix translation is not required anymore as you correctly said. I need to delete it from the